recipe/meta.yaml

```diff
@@ -4,7 +4,7 @@
     "atomic", "charconv", "chrono", "cobalt", "container", "context", "contract", "coroutine",
     "date_time", "filesystem", "graph", "iostreams", "locale", "log", "log_setup",
     "math_c99", "math_c99f", "math_tr1", "math_tr1f", "prg_exec_monitor",
-    "program_options", "random", "regex", "serialization", "system", "thread",
+    "program_options", "random", "regex", "serialization", "stacktrace", "system", "thread",
     "timer", "type_erasure", "unit_test_framework", "wave", "wserialization"
 ] %}
 {% set boost_libs = boost_libs + ["math_c99l", "math_tr1l"] %}  # [not ppc64le]
```

Comment:
You cannot add this to the overall `boost_libs` because these are expected to exist on all platforms (and you're not enabling it on windows). Could be something like

```suggestion
    "program_options", "random", "regex", "serialization", "system", "thread",
    "timer", "type_erasure", "unit_test_framework", "wave", "wserialization"
] %}
# there's no backtrace built on windows in conda-forge yet
{% set boost_libs = boost_libs + ["stacktrace"] %}              # [not win]
{% set boost_libs = boost_libs + ["math_c99l", "math_tr1l"] %}  # [not ppc64le]
```